src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,47 @@
+package edu.kis.vh.nursery;
+
+
+
+public class DefaultCountingOutRhymer {
+
+
+    private static final int MAX_CAP = 12;
+    private static final int EMPTY_STACK = -1;
+    private static final int LAST_ELEM = 11;
+    private final int[] numbers = new int[MAX_CAP];
+    private static final int IVALID_STACK = -1;
+    private int total = EMPTY_STACK;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.